Dockerfile

```
@@ -7,7 +7,7 @@ ARG GO_IPFS_VER
 RUN apt-get update -q && apt-get install -y git curl gnupg jq build-essential gawk zip tzdata && \
     ln -fs /usr/share/zoneinfo/Etc/UTC /etc/localtime && \
     dpkg-reconfigure --frontend noninteractive tzdata
-RUN curl -s "https://dist.ipfs.tech/go-ipfs/${GO_IPFS_VER}/go-ipfs_${GO_IPFS_VER}_linux-amd64.tar.gz" | tar vzx -C /usr/local/bin/ go-ipfs/ipfs --strip-components=1
+RUN curl --retry 5 -s "https://dist.ipfs.tech/go-ipfs/${GO_IPFS_VER}/go-ipfs_${GO_IPFS_VER}_linux-amd64.tar.gz" | tar vzx -C /usr/local/bin/ go-ipfs/ipfs --strip-components=1
```

I resolved the conflict too hastily. 

```suggestion
RUN curl --retry 5 \
    "https://dist.ipfs.tech/go-ipfs/${GO_IPFS_VER}/go-ipfs_${GO_IPFS_VER}_linux-amd64.tar.gz"  -o /tmp/ipfs.tar.gz && \
    tar vzx -f /tmp/ipfs.tar.gz -C /usr/local/bin/ go-ipfs/ipfs --strip-components=1
```